quipucords/api/fingerprint/serializer.py

```diff
@@ -14,14 +14,13 @@
 from rest_framework.serializers import (IntegerField,
                                         CharField,
                                         ChoiceField,
-                                        UUIDField,
                                         DateField,
-                                        NullBooleanField)
+                                        NullBooleanField,
+                                        ModelSerializer)
 from api.models import SystemFingerprint
-from api.common.serializer import NotEmptySerializer
 
 
-class FingerprintSerializer(NotEmptySerializer):
+class FingerprintSerializer(ModelSerializer):
     """Serializer for the Fingerprint model."""
 
     os_name = CharField(required=True, max_length=64)
```

Comment:
os_version should have required=False based on the model change made above.

Reply:
done